include/miopen/miopen.h

```diff
@@ -4879,6 +4879,15 @@ miopenStatus_t miopenSetFindOptionPreallocatedTensor(miopenFindOptions_t options
                                                      miopenTensorArgumentId_t id,
                                                      void* buffer);
 
+/*! @brief Forces library to attach kernel binaries to solutions for later saving. This allows zero
+ * lookup miopenRunSolution calls after miopenLoadSolution. Default value is false.
+ *
+ * @param options    Options object to update
+ * @param attach     True means attaching, false - skipping
+ * @return           miopenStatus_t
+ */
+miopenStatus_t miopenSetFindOptionAttachBinaries(miopenFindOptions_t options, bool attach);
```

Comment:
I suggest using a mask to set/unset boolean options. This will allow the API to be extended in the future without adding new functions.
```suggestion
miopenStatus_t miopenSetFindOptionsBool(miopenFindOptions_t options, uint32_t set, uint32_t unset);
```

Reply:
I'd rather keep explicit functions for separate options, since they are unrelated. Plus, with the current approach each option can actually be extended to be a bitmask itself, if needed. For example, if we would want to add another mode of attaching binaries, we could just use a value of 2 for that.

Reply:
I'd prefer a simple but powerful API instead of a dozen functions, something like OpenCL API. You could reserve 2 or more bits for this option. Anyway, it's up to you.

Reply:
> I'd rather keep explicit functions for separate options, since they are unrelated. Plus, with the current approach each option can actually be extended to be a bitmask itself, if needed.

Yes I think the explicit function is much better than using bitmasks.